packages/creator/src/lib/helpers/cachedFetchLike/cachedFetchLike.ts

```diff
@@ -0,0 +1,46 @@
+const defaultCache = new Map<string, string>();
+
+const isArianeeSchema = (url: string) =>
+  url.startsWith('https://cert.arianee.org/version');
```

Comment:
We should also cache https://cert.arianee.org/contractAddresses

Reply:
yes adding it